libvast/src/detail/backtrace.cpp

```diff
@@ -0,0 +1,93 @@
+/******************************************************************************
+ *                    _   _____   __________                                  *
+ *                   | | / / _ | / __/_  __/     Visibility                   *
+ *                   | |/ / __ |_\ \  / /          Across                     *
+ *                   |___/_/ |_/___/ /_/       Space and Time                 *
+ *                                                                            *
+ * This file is part of VAST. It is subject to the license terms in the       *
+ * LICENSE file found in the top-level directory of this distribution and at  *
+ * http://vast.io/license. No part of VAST, including this file, may be       *
+ * copied, modified, propagated, or distributed except according to the terms *
+ * contained in the LICENSE file.                                             *
+ ******************************************************************************/
+
+#include "vast/detail/backtrace.hpp"
+
+#include "vast/config.hpp"
+
+#if !VAST_ENABLE_BACKTRACE
+void backtrace(){};
+#else
+#  if __has_include(<execinfo.h>)
```

Comment:
We could also expose the used mechansim via CMake. I don't think the additional time investment is worth it right now though.

Reply:
I agree. We do not need this currently, and detecting this via CMake is a lot of work.